Charts: update the 'Dynamic Series from the DataSource' demo description (#1866)
DevExpress/devextreme-demos: JSDemos/Demos/Charts/SeriesTemplates/description.md

> In some scenarios, you need to add more series to the data source after you created the Chart. In this case, organize your data source structure as follows:

Reviewer: ```suggestion
In certain scenarios, you may need to add more series to the data source after you created the Chart. In this case, arrange your data source structure as follows:
```

    [
        {seriesName: series1, arg: arg11Value, val: value11 }
        {seriesName: series1, arg: arg12Value, val: value12 }
        ...
        {seriesName: seriesM, arg: argM1Value, val: valueM1 }
        {seriesName: seriesM, arg: argM2Value, val: valueM2 }
        ...
    ]

> Each object in the data source must represent a point in a single series.

Reviewer: Never use the word "represent". **Under no circumstances this word can be in our documentation**. This is utterly important. Please refer to the following guideline topic for details: https://tw.devexpress.devx/language-guidelines/misused-words/represent/

```suggestion
Every object in the data source should correspond to a point in a single series.
```

> This demo uses the structure displayed above to organize the data:

Reviewer: The noun "data" is uncountable, therefore, you should not use an article with it unless you have a very strong reason to do so. In this case, the definite article places extra emphasis on data. I do not feel that we need it here.

```suggestion
This demo uses the structure displayed above to organize data:
```

    {
        year: 1970,
        country: 'Saudi Arabia',
        oil: 192.2,
    }

> To define series, use the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object to specify common settings for all series: the [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/#argumentField), the [valueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/#valueField) and the [type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/#type).

Reviewer: Mind the Oxford comma: https://tw.devexpress.devx/punctuation/#use-comma

```suggestion
To define series, use the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object to specify common settings for all series: the [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/#argumentField), the [valueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/#valueField), and the [type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/#type).
```

Then, use the [seriesTemplate](/Documentation/ApiReference/UI_Components/dxChart/Configuration/seriesTemplate/) configuration object to define a template for the series. Within this object, assign the data source field that specifies the series name to the [nameField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/seriesTemplate/#nameField) property.

> If you need to specify individual values for properties of a particular series, assign a callback function to the [customizeSeries](/Documentation/ApiReference/UI_Components/dxChart/Configuration/seriesTemplate/#customizeSeries) property of the [seriesTemplate](/Documentation/ApiReference/UI_Components/dxChart/Configuration/seriesTemplate/) object. This demo uses the [customizeSeries](/Documentation/ApiReference/UI_Components/dxChart/Configuration/seriesTemplate/#customizeSeries) function to display a line instead of a bar for the `year: 2009`.

Reviewer: Is this an expression? `year: 2009`

If it is, then you do not need an article.

Author: Yes, it is an expression. Noted
